integ/workflow/persistence_loading_policy/routers.go

```diff
@@ -0,0 +1,191 @@
+package persistence_loading_policy
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/integ/workflow/common"
+	"github.com/indeedeng/iwf/integ/workflow/persistence"
+	"github.com/indeedeng/iwf/service"
+	"github.com/indeedeng/iwf/service/common/ptr"
+	"log"
+	"net/http"
+)
+
+const (
+	WorkflowType                  = "persistence_loading_policy"
+	State1                        = "S1"
+	State2                        = "S2"
+	LoadingTypeAll                = "all"
```

Comment:
I think it's better to reuse the enum from idl. if you type 
```
iwfidl.ALL_WITHOUT_LOCKING
iwfidl.PARTIAL_WITHOUT_LOCKING
iwfidl.PARTIAL_WITH_EXCLUSIVE_LOCK
```

In golang, you can convert the enum from/to string directly:

```
  strVal = string(iwfidl.ALL_WITHOUT_LOCKING)

  persistenceLoadingType = iwfidl.PersistenceLoadingType(strVal)
```

Reply:
Great!